test/server/cards/01_SOR/events/ForcedSurrender.spec.ts

```diff
@@ -0,0 +1,147 @@
+describe('Forced Surrender', function() {
+    integration(function(contextRef) {
+        describe('Forced Surrender\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['forced-surrender', 'atst', 'atst'],
+                        groundArena: ['wampa'],
+                        discard: ['daring-raid'],
+                        leader: 'sabine-wren#galvanized-revolutionary'
+                    },
+                    player2: {
+                        groundArena: ['maz-kanata#pirate-queen'],
+                        hand: ['atst', 'atst']
+                    }
+                });
+            });
+
+            it('draws 2 cards and makes opponents whose base you damaged this phase discard 2 cards', function () {
+                const { context } = contextRef;
+
+                const reset = () => {
+                    if (context.mazKanata.location === 'discard') {
+                        context.player2.moveCard(context.mazKanata, 'ground arena');
+                    }
+                    context.player2.claimInitiative();
+
+                    context.player1.hand.forEach((card) => context.player1.moveCard(card, 'deck'));
+                    context.player2.hand.forEach((card) => context.player2.moveCard(card, 'deck'));
+                    context.player1.moveCard(context.forcedSurrender, 'hand');
+                    context.player1.passAction();
+
+                    context.player2.clickPrompt('Done');
+                    context.player1.clickPrompt('Done');
+                };
+
+                context.player1.passAction();
+
+                // Scenario 1: Damage dealt with direct attack
+                context.player2.clickCard(context.mazKanata);
+                context.player2.clickCard(context.p1Base);
+
+                context.player1.clickCard(context.wampa);
+                context.player1.clickCard(context.p2Base);
+
+                context.player2.passAction();
+
+                context.player1.clickCard(context.forcedSurrender);
+                expect(context.player2).toBeActivePlayer();
+                expect(context.player1.handSize).toBe(4);
+                expect(context.player2.handSize).toBe(0);
+
+                reset();
+
+                // Scenario 2: No damage dealt this phase
+                context.player2.clickCard(context.mazKanata);
+                context.player2.clickCard(context.p1Base);
+
+                context.player1.clickCard(context.forcedSurrender);
+                expect(context.player2).toBeActivePlayer();
+                expect(context.player1.handSize).toBe(4);
+                expect(context.player2.handSize).toBe(2);
+
+                reset();
+
+                // Scenario 3: Damage dealt with overwhelm
+                context.player2.clickCard(context.mazKanata);
```

Comment:
What's the reason for doing the Maz Kanata attack every time?

Reply:
Just to ensure that we have damaged dealt to both bases.

Reply:
Still a little unclear - what is that specifically testing? Feels like we could do it one time in the case where we have dealt no damage to opponent base and should be good